projects/plugins/jetpack/extensions/blocks/donations/edit.js

```diff
@@ -66,16 +69,19 @@ const Edit = props => {
 				'1 month': -1,
 				'1 year': -1,
 			} );
+			unlockPostSaving( 'donations' );
 			return;
 		}
 
 		// Only create products if we have the correct plan and stripe connection.
 		fetchDefaultProducts( currency ).then( defaultProducts => {
-			return setProducts( filterProducts( defaultProducts ) );
+			setProducts( filterProducts( defaultProducts ) );
+			unlockPostSaving( 'donations' );
 		}, apiError );
 	};
 
 	useEffect( () => {
+		lockPostSaving( 'donations' );
 		const updateData = () => fetchStatus( 'donation' ).then( mapStatusToState, apiError );
 		updateData();
 	}, [ currency ] );
```

Comment:
We need to update the dependencies of this effect:

```
ESLint: React Hook useEffect has a missing dependency: 'lockPostSaving'. Either include it or remove the dependency array.(react-hooks/exhaustive-deps)
```

Reply:
Updated